index.html

```diff
@@ -9,51 +9,229 @@
 </head>
 <body>
     <!--Head Section-->
-    <header>
-        <div class="logo">
-            <a class="a_logo" href="#">My Logo</a>
+   <div class="head-section">
+        <header>
+            <div class="logo">
+                <a class="a_logo" href="#">My Logo</a>
+            </div>
+            <nav class="menu">
+                <a href="#">
+                    <img src="./images/Icon-Menu.png" alt="Menu" class="menu-icon"/>
+                </a>
+            </nav>
+        </header>
+        <!--Main Section-->
+        <main>
+            <h1 class="intro">I'm Ernest Glad to see you!</h1>
+            <p class="main-paragraph">I am a Web developer! I can help you build your website. Kindly go through my work samples! If you are interested and you have any website to develop, do contact me. I would appreciate recommendation as well.</p>
+            <p class="connect">LET'S CONNECT</p>
+        </main>
+        
+        <!--Headline Section-->
+        <div class="headline-section">
+            <ul>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/twiter.png" alt="Twitter"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social linkendIn">
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+   </div>
+   
+    <!--Work Section-->
+    <section class="work-section">
+        <section class="tonic-frontpage padding">
+            <div class="frontpage-icon">
+                <img src="./images/Snapshoot-Portfolio.png" alt="Portfolio">
+            </div>
+            <div class="tonic">
+                <h2>Tonic</h2>
+                <p class="tonic-section-header">
+                    <span class="canopy">CANOPY</span> <img src="./images/Counter.png" alt="counter" class="counter"/>
+                    <span class="back-end-dev">Back End Dev</span> <img src="./images/Counter.png" alt="counter"/>
+                    <span class="year">2022</span>
+                </p>
```

Comment:
Let's make this a list, and each item as a list item. Don't you think that will make more sense since this is a list